src/Authentication/appsettings.json

```diff
@@ -16,7 +16,7 @@
     "MaskinportenWellKnownAlternativeConfigEndpoint": "https://test.maskinporten.no/.well-known/oauth-authorization-server",
     "IdPortenWellKnownConfigEndpoint": "https://oidc-ver2.difi.no/idporten-oidc-provider/.well-known/openid-configuration",
     "IdPortenAlternativeWellKnownConfigEndpoint": "https://oidc-test.difi.no/idporten-oidc-provider/.well-known/openid-configuration",
-    "OpenIdWellKnownEndpoint": "http://localhost:5040/authentication/api/v1/openid",
+    "OpenIdWellKnownEndpoint": "http://localhost:5040/authentication/api/v1/openid/.well-known/openid-configuration",
```

Comment:
Was this a neccessary change for dev-only?

Based on the values in values.yaml for authentication which sets this config value for the environments it's the issuer not actually the well-known endpoint.

Reply:
ya right, I had to change that for dev only. I later moved all my changes to appsettings.Development.json for development. I will revert this